parser/dns.go

```diff
@@ -6,20 +6,43 @@ import (
 	"github.com/activecm/rita/parser/parsetypes"
 	"github.com/activecm/rita/pkg/data"
 	"github.com/activecm/rita/pkg/hostname"
+
+	log "github.com/sirupsen/logrus"
 )
 
-func parseDNSEntry(parseDNS *parsetypes.DNS, filter filter, retVals ParseResults) {
+func parseDNSEntry(parseDNS *parsetypes.DNS, filter filter, retVals ParseResults, logger *log.Logger) {
 
 	// extract and store the dns client ip address
 	src := parseDNS.Source
 	srcIP := net.ParseIP(src)
 
+	// verify that ip address was parsed successfully
```

Comment:
PR #788 introduces a dependency on the `dstIP` as well in the DNS parser. 
https://github.com/activecm/rita/pull/788/files#diff-819801100fce036a714f6949e25d050e069ebe2a7448cbaed5618e091de6bc31R17

Please add a check here similar to the one in the conn log parser to check the `dstIP` as well.